### DISCUSSION TITLE
WebXR Zoom Percentage

### DISCUSSION THREAD

**Discussion by sprengerst** [google/model-viewer#2754] [Ideas]
### Description

Hi,

if a project is loaded with WebXR there is no zoom percentage shown if a model is scaled. In QuickLook and SceneViewer a zoom percentage is shown.
 
Is there any configuration option to enable a zoom percentage text also in WebXR?

Best,
Stefan

### Version

- model-viewer: latest

### Browser Affected

- [X ] Chrome, version: xx.x.xxxx.xx
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [X ] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR

- [X ] WebXR
- [X ] SceneViewer
- [ X] QuickLook

**Comment by elalish** [comment]
Not currently. We could expose an event with the scale value when it changes. We opted for a snap to 100% instead since we couldn't think of a use case for knowing the  specific % other than checking it if  was true scale or not. Do you have a use case?

**Reply by sprengerst** [reply]
I think it should be possible (or added as an event) because it‘s also a standard in QuickLook. To achieve a homogeneous look and feel across all devices and to replace scene-viewer. 

A use case could be to show a text that a product has reached „true scale“ while zooming when 100% is hit. As an example with paintings, posters, photographs.

**Reply by elalish** [reply]
Our goal is not to achieve a homogeneous look with Quick Look; that is already impossible due to their small API and no glTF support. How about [`ar-scale = "fixed"`](https://modelviewer.dev/docs/index.html#entrydocs-augmentedreality-attributes-arScale) for your use case?
